runtime/common/src/parachains.rs

```diff
@@ -147,10 +141,6 @@ const EMPTY_TRIE_ROOT: [u8; 32] = [
 
 /// Total number of individual messages allowed in the parachain -> relay-chain message queue.
 const MAX_QUEUE_COUNT: usize = 100;
-/// Total size of messages allowed in the parachain -> relay-chain message queue before which no
-/// further messages may be added to it. If it exceeds this then the queue may contain only a
-/// single message.
-const WATERMARK_QUEUE_SIZE: usize = 20000;
```

Comment:
actually, this should be restored along with the logic in `check_upward_messages` which used it.